exercises/simple-cipher/src/test/java/SimpleCipherStepThreeTest.java

```diff
@@ -1,63 +0,0 @@
-import org.junit.Ignore;
```

Comment:
Is it necessary to rename the test files? I think for consistency's sake we should stick to the previous names. Thoughts @FridaTveit ?